ios/Cove/NewWallet/NewWalletSelectView.swift

```diff
@@ -21,7 +19,7 @@ struct NewWalletSelect: View {
                 .padding(.bottom, 20)
                 .padding(.horizontal, 30)
             Spacer()
-            Button(action: { mainViewModel.pushRoute(RouteFactory().newHotWallet()) }) {
+            NavigationLink(value: RouteFactory().newHotWallet()) {
```

Comment:
@justinmoon this is really nice, declarative routing, you just pass the route you want it to go to.